erpnext_shopify/erpnext_shopify/doctype/shopify_settings/shopify_settings.js

```diff
@@ -35,21 +35,19 @@ frappe.ui.form.on("Shopify Settings", "refresh", function(frm){
 		frm.toggle_reqd("delivery_note_series", true);
 		frm.toggle_reqd("customer_group", true);
 		
-		if (frm.doc.last_sync_datetime) {
-			cur_frm.add_custom_button(__('Sync Shopify'),
-				function() {
-					frappe.call({
-						method:"erpnext_shopify.api.sync_shopify",
-						freeze: true,
-						freeze_message: "Syncing",
-						callback:function(r){
-							if(!r.exc){
-								frappe.msgprint(__("Sync Completed!!"))
-							}
+		cur_frm.add_custom_button(__('Sync Shopify'),
+			function() {
+				frappe.call({
+					method:"erpnext_shopify.api.sync_shopify",
+					freeze: true,
+					freeze_message: "Syncing",
+					callback:function(r){
+						if(!r.exc){
+							frappe.msgprint(__("Sync Completed!!"))
```

Comment:
As the task is queued in a long lob, it should not be freezed and remove the message.